cli/generators/crud/crudGenerator.js

```diff
@@ -0,0 +1,23 @@
+var Generator = require('yeoman-generator');
+var ejs = require('ejs');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  appendTpl(templatePath, destinationPath, data) {
+    var fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
```

Comment:
Hello! I found it [this](https://stackoverflow.com/questions/19178523/can-yeoman-generators-update-existing-files) article. But now that you mention, I'll take a further read to what [fs](https://github.com/SBoudrias/mem-fs-editor) can do. thanks to bring this to the table again.